lib/matplotlib/tests/test_axes.py

```diff
@@ -725,6 +725,42 @@ def test_polycollection_joinstyle():
     ax.set_ybound(0, 3)
 
 
+@raises(ValueError)
```

Comment:
I have no clue why this doesn't fail for all the builds.
Raises is not defined. I think you need to use the "new" pytest way of doing this:

```python
def test_fill between_2d_x_input():
    …
    with pytest.raises(ValueError):
         ax.plot(…)
```